app/views/unattended/poap/POAP.erb

```diff
@@ -0,0 +1,9 @@
+<%#
```

Comment:
It looks like this would belong in docs rather than a default template. Assuming Cisco's are proprietary, do you mind maybe submitting a sample script of yours in [community-templates](https://github.com/theforeman/foreman/pull/2464/)?